# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: thousand separator for Swiss locales is '
# DP: Related bugs: #345481
# DP: Dpatch author: Denis Barbier <barbier@debian.org>
# DP: Patch author: Denis Barbier
# DP: Upstream status: not submitted
# DP: Date: 2006-02-16

Index: glibc-2.3.6/localedata/locales/de_CH
===================================================================
--- glibc-2.3.6.orig/localedata/locales/de_CH
+++ glibc-2.3.6/localedata/locales/de_CH
@@ -62,7 +62,7 @@
 int_curr_symbol           "<U0043><U0048><U0046><U0020>"
 currency_symbol           "<U0046><U0072><U002E>"
 mon_decimal_point         "<U002E>"
-mon_thousands_sep         "<U0020>"
+mon_thousands_sep         "<U0027>"
 mon_grouping              3;3
 positive_sign             ""
 negative_sign             "<U002D>"
@@ -78,7 +78,7 @@
 
 LC_NUMERIC
 decimal_point             "<U002E>"
-thousands_sep             "<U0020>"
+thousands_sep             "<U0027>"
 grouping                  3;3
 END LC_NUMERIC
 
Index: glibc-2.3.6/localedata/locales/it_CH
===================================================================
--- glibc-2.3.6.orig/localedata/locales/it_CH
+++ glibc-2.3.6/localedata/locales/it_CH
@@ -57,9 +57,7 @@
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point             "<U002C>"
-thousands_sep             "<U0027>"
-grouping                  3;3
+copy  "de_CH"
 END LC_NUMERIC
 
 LC_TIME
